portfolio/src/main/webapp/index.html

```diff
@@ -58,6 +58,8 @@ <h1>Mauricio's Portfolio</h1>
         <br>
```

Comment:
This does not impact current code changes, just a suggestiobn. Use "placeholder" for "ex. Nice color choices! However, try to have more contrast." That way user does not have to clear the text already in  the text box before typing.